### PR TITLE
Let the validation error strings are translatable.

### PR DESCRIPTION
**Pull request by jsuchome**
This is just one string of many (the one I've met in https://github.com/crowbar/crowbar-openstack/pull/201) and I think others should be moved to locale file as well.

Or is there any reason we do not have it? @tserong @mjura

### CODE REVIEW

**Comment by aplanas** [comment]
+1

**Comment by tboerger** [comment]
:+1:

File: crowbar_framework/config/locales/ceph/en.yml [hunk 1]
```diff
@@ -43,3 +43,5 @@ en:
             enabled: 'Protocol'
             certfile: 'SSL Certificate File'
             keyfile: 'SSL (Private) Key File'
+      validation:
+        ses_repos: 'The SUSE Enterprise Storage repositories have not been setup.'
```
Comment (vuntz) [inline crowbar_framework/config/locales/ceph/en.yml:51]: So, it's not introduced by your commit, but I think the error string should be made vendor-neutral. Something like "Ceph is not available in the repositories.", and then we would patch the string in the package to mention SES.

**Comment by vuntz** [comment]
+1; I'm happy to do a followup PR for my comment.

**Comment by jsuchome** [comment]
I've added more strings from ceph_service.rb

**Comment by tserong** [comment]
LGTM

**Comment by nkrinner** [comment]
+1

File: crowbar_framework/config/locales/ceph/en.yml [hunk 2]
```diff
@@ -43,3 +43,11 @@ en:
             enabled: 'Protocol'
             certfile: 'SSL Certificate File'
             keyfile: 'SSL (Private) Key File'
+      validation:
+        disk_missing: "Nodes %{nodes} for ceph-osd role are missing at least one unclaimed disk larger than %{size}GB."
```
Comment (tboerger) [inline crowbar_framework/config/locales/ceph/en.yml]: Please use the single quotes like it's done for all other keys.

**Comment by tboerger** [comment]
After making the quotes more unified you get my :+1:

**Comment by vuntz** [comment]
+1

**Comment by dirkmueller** [comment]
This commit apparently causes https://bugzilla.suse.com/show_bug.cgi?id=967255

**Pull request merged**